products/alibaba-dragonwell.md

```diff
@@ -0,0 +1,87 @@
+releaseDateColumn: true
+
+# There is one repository for each major LTS release.
+# And yes, tagging at Alibaba is a mess !
+auto:
+-   git: "https://github.com/dragonwell-project/dragonwell8.git"
+    regex: '^dragonwell-standard-(?<version>[\d\.\+]+)_jdk.+-ga$'
+    template: '{{version}}'
+-   git: "https://github.com/dragonwell-project/dragonwell8.git"
+    regex: '^dragonwell-(?<version>[\d\.\+]+)_jdk.+-ga$'
+    template: '{{version}}'
+-   git: "https://github.com/dragonwell-project/dragonwell8.git"
+    regex: '^dragonwell-(?<version>[\d\.\+]+)-GA$'
+    template: '{{version}}'
+-   git: "https://github.com/dragonwell-project/dragonwell8.git"
+    regex: '^v(?<version>[\d\.\+]+)-GA$'
+    template: '{{version}}'
+-   git: "https://github.com/dragonwell-project/dragonwell11.git"
+    regex: '^dragonwell-standard-(?<version>[\d\.\+]+)_jdk.+-ga$'
+    template: '{{version}}'
+-   git: "https://github.com/dragonwell-project/dragonwell11.git"
+    regex: '^dragonwell[-_](?<version>[\d\.\+]+)_.+-ga$'
+    template: '{{version}}'
+-   git: "https://github.com/dragonwell-project/dragonwell17.git"
+    regex: '^dragonwell-standard-(?<version>17[\d\.\+]+)_jdk.+-ga$'
+    template: '{{version}}'
+-   git: "https://github.com/dragonwell-project/dragonwell17.git"
+    regex: '^dragonwell-(?<version>17[\d\.\+]+)_jdk.+-ga$'
+    template: '{{version}}'
+-   git: "https://github.com/dragonwell-project/dragonwell17.git"
+    regex: '^jdk-(?<version>17[\d\.\+]+)-ga$'
+    template: '{{version}}'
+
+# Do not forget to update the "auto" configuration on each new major release.
+# End of public updates available on https://github.com/dragonwell-project/dragonwell17/wiki/Alibaba-Dragonwell-Support
+releases:
+# First official release is 17.0.1.
+-   releaseCycle: "17"
+    lts: true
+    releaseDate: 2021-10-18
+    eol: 2029-09-30
+    latest: "17.0.6.0.6+9"
+    latestReleaseDate: 2023-02-02
+
+# First official release is 11.0.7.2.
+-   releaseCycle: "11"
+    lts: true
+    releaseDate: 2020-04-24
+    eol: 2027-09-30
+    latest: "11.0.18.14"
+    latestReleaseDate: 2023-02-06
+
+-   releaseCycle: "8"
+    lts: true
+    releaseDate: 2019-06-24
+    eol: 2026-06-30
+    latest: "8.14.15"
+    latestReleaseDate: 2023-01-31
+    link: https://github.com/dragonwell-project/dragonwell8/wiki/Alibaba-Dragonwell8-Standard-Edition-Release-Notes
+
+---
+
+> [Alibaba Dragonwell](https://dragonwell-jdk.io/) is a [GPLv2 with CPE](https://openjdk.java.net/legal/gplv2+ce.html)
+> licensed build of the Open Java Development Kit (OpenJDK) with long-term support and patches from
+> Alibaba. It is available in two editions : a Standard Edition for general usage and an Extended
+> Edition optimized for the Alibaba cloud. Both editions are certified using the [AQAvit quality
+> verification suite](https://adoptium.net/aqavit/). It is available on Linux, Windows and Docker.
+
+Alibaba only provide support for Dragonwell builds based on LTS OpenJDK releases (8, 11, 17, 21...).
```

Comment:
This is slightly confusing - are non-LTS builds provided, but unsupported, or not provided at all?

```suggestion
Alibaba only provides support for Dragonwell builds based on LTS OpenJDK releases (8, 11, 17, 21...).
```

Reply:
Not provided at all.

Reply:
Changed wording slightly to drop the word "support" to clarify this.